src/foam/nanos/controller/ApplicationController.js

```diff
@@ -813,6 +814,12 @@ foam.CLASS({
       });
     },
 
+    async function login(identifier, password) {
```

Comment:
Given that we are trying to move stuff out of ApplicationController, why is this needed?

Reply:
Some methods on the controller are needed to be called right after login but they are not exported and currently we were just importing `ctrl` then call them manually. Bundle them in a single method so that it doesn't need to call these un-exposed methods.